.github/workflows/comment-wheels-pr.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     if: github.event.workflow_run.event == 'pull_request' && github.event.workflow_run.conclusion == 'success'
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/github-script@v6
+      - uses: actions/github-script@v7
```

Comment:
shouldn't this be detected by our bot?

Reply:
maybe? This was also a breaking change, so maybe the bot does not open PR in this case?